### PR TITLE
t9 case

### PR DESCRIPTION
**Pull request by EditaM94**

### CODE REVIEW

File: cypress/integration/login.js [hunk 1]
```diff
@@ -6,40 +11,42 @@ const roles= [['User',1],['Team Lead',2], ['Manager',5], ['Accountant',5],['Admi
 describe('Login functionality', function() {
     
     it('Should display validation for empty user after attempted loggin', function () {
-        cy.visit('/')
-        cy.get('.Select.not-valid').should('not.visible')
-        cy.get('[type="submit"]').click()
-        cy.get('.Select.not-valid').should('be.visible')
+        loginPage.visit()
+        loginPage.getvalidateInput().should('not.visible')
```
Comment (MonkeviciuteGerda) [inline cypress/integration/login.js]: Use camel case, please, e.g. getValidateInput, getSubmitButton

File: cypress/objects/calendarPage.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    getTodayDate(){
+        return cy.get('.calendar--today')
+    }
+
+    getUserInfo(){
+        return cy.get('.user-info__title')
+    }
+
+    getMenuItemsCount(){
```
Comment (MonkeviciuteGerda) [inline cypress/objects/calendarPage.js]: Does it get count, or just all the elements that match the selector?

File: cypress/objects/loginPage.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+    getUserSelect(){
+        return cy.get('[aria-label="TestCon User 5"]')
+    }
+
+    getLoginRoleSelect(){
+        return cy.get('[id="loginForm.role"]')
+    }
+
+    getRoleLead(){
```
Comment (MonkeviciuteGerda) [inline cypress/objects/loginPage.js]: What is this? Could you use getAnyRole method with a parameter instead?

File: cypress/integration/login.js [hunk 4]
```diff
@@ -6,40 +11,42 @@ const roles= [['User',1],['Team Lead',2], ['Manager',5], ['Accountant',5],['Admi
...
     it('Should be able to login with role User', function () {
-        cy.get('[id="loginForm.userId"]').click({force:true})
-        cy.get('[aria-label="TestCon User 5"]').click()
-        cy.get('[id="loginForm.role"]').click({force:true})
-        cy.get('[aria-label="Team Lead"]').click()
-        cy.get('[type="submit"]').click()
+        loginPage.getUserInput().click({force:true})
+        loginPage.getUserSelect().click()
+        loginPage.getLoginRoleSelect().click({force:true})
+        loginPage.getRoleLead().click()
```
Comment (MonkeviciuteGerda) [inline cypress/integration/login.js]: What is this?

**Review by MonkeviciuteGerda — CHANGES_REQUESTED** [review]
Very good job! Minor updates required

**Comment by EditaM94** [comment]
Updated

File: cypress/objects/loginPage.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    getSubmitButton(){
+        return cy.get('[type="submit"]')
+    }
+
+    getUserInputField(){
+        return cy.get('[id="loginForm.userId"]')
+    }
+
+    getMyUserSelected(){
+        return cy.get('[aria-label="TestCon User 5"]')
```
Comment (MonkeviciuteGerda) [inline cypress/objects/loginPage.js:20]: Funny :D It would be best to pass as a parameter just like with the role

**Review by MonkeviciuteGerda — APPROVED** [review]

**Pull request merged**